src/game/option.c

```diff
@@ -566,7 +569,8 @@ void DoPassportOption(INVENTORY_ITEM *inv_item)
         1,
     };
 
-    if (InputDB.left && (InvMode != INV_DEATH_MODE || SavedGamesCount)) {
+    if (InputDB.left
+        && (InvMode != INV_DEATH_MODE || SavedGamesCount || page > 1)) {
```

Comment:
This is sketchy, can we simplify it?

Reply:
This was taking from Tomb2Main. Possibly? Why is it sketchy?